priv/kafka.bnf

```diff
@@ -250,3 +250,9 @@ DescribeGroupsResponse => [GroupDescription]
     MemberMetadata => bytes
     MemberAssignment => bytes
 
+SaslHandshakeRequest => Mechanism
+  Mechanism => string
+
+SaslHandshakeResponse => ErrorCode [EnabledMechanism]
+  ErrorCode => int16
+  EnabledMechanism => string
```

Comment:
Yeah, keep it for now. This naming convention is derived from  kafka 0.8 spec. 
We'll change naming convention in 0.9 or maybe make it 1.0